### ISSUE TITLE
Error with the tab exemple

### ISSUE THREAD

**Issue by pascalr0410** [lovasoa/SQLpage#58]
Hello

**index.sql is a c/c of example given in [doc](https://sql.ophir.dev/documentation.sql?component=tab#component) :**

```
SELECT 
    'tab' as component;
SELECT 
    'My First tab' as title,
    1 as active;
SELECT 
    'This is tab two' as title;
SELECT 
    'Third tab is crazy' as title;
```

**which causes the following error :**

```
Sorry, but we were not able to process your request. 

Error:

 Failed to create a render context from the header context.

Caused by:
    0: The component 'tab' was not found.
    1: Couldn't load "sqlpage/templates/tab.handlebars" into cache
    2: Unable to read local file "sqlpage/templates/tab.handlebars"
    3: No such file or directory (os error 2)
```

**Version 0.9.4 with Windows and linux**

Regards,
Pascal

**Comment by lovasoa** [comment]
![image](https://github.com/lovasoa/SQLpage/assets/552629/76792d05-cb22-4838-83ab-483f808aeb92)


v0.9.5 still hasn't be released ! You are probably using v0.9.4.

If you want to test nightly versions, you can use the [official docker image](https://hub.docker.com/r/lovasoa/sqlpage). The `latest` tag on the docker image always points to the latest unpublished changes. 

```
docker run -it --name sqlpage -p 8080:8080 --volume "$(pwd):/var/www" --rm lovasoa/sqlpage
```


Don't hesitate to report bugs in the work-in-progress version.

**Comment by lovasoa** [comment]
I just released [v0.9.5](https://github.com/lovasoa/SQLpage/releases/tag/v0.9.5) which contains the new tab component.
